ci-operator/step-registry/openshift-qe/winc/openshift-qe-winc-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
```

Comment:
If you move the ref to a folder in `ci-operator/step-registry/cucushift/installer/rehearse/` (most of workflow from QE are located there), then we do not need to wait the approval from openshift-qe organization (mainly used by OpenShift PerfScale team).

Reply:
@liangxia The thing is, that this openshift-qe-winc-ref is going to be used in all WINC jobs, that is why I can't simply add it under ci-operator/step-registry/cucushift/installer/rehearse/aws folder. Could I do something similar as hypershift team did? Create a winc-extended folder under [ci-operator/step-registry/cucushift](https://github.com/openshift/release/tree/master/ci-operator/step-registry/cucushift) ? Would there be any problem into that? This way, all the extra refs required for winc team (there will be more) could  be included there.

Reply:
Fine to me to put it into the folder `ci-operator/step-registry/cucushift`. No need to add  `-extended`.